File: SoloLearn/Code Coach/Roadrunner/Roadrunner.java
import java.util.Scanner;

public class Roadrunner {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int distance = input.nextInt();
        input.nextLine();
        int runnerSpeed = input.nextInt();
        input.nextLine();
        int coyoteSpeed = input.nextInt();
        double runnerTime = distance / runnerSpeed, coyoteTime = (distance + 50) / coyoteSpeed;
        if (runnerTime < coyoteTime)
            System.out.println("Meep Meep");
        else
            System.out.println("Yum");
        input.close();
    }
} 

File: SoloLearn/Code Coach/Roadrunner/Roadrunner.c
#include <stdio.h>

int main() {
    int distance, runnerSpeed, coyoteSpeed;
    scanf("%d", &distance);
    scanf("%d", &runnerSpeed);
    scanf("%d", &coyoteSpeed);
    double runnerTime = distance / runnerSpeed, coyoteTime = (distance + 50) / coyoteSpeed;
    if (runnerTime < coyoteTime)
        printf("Meep Meep");
    else
        printf("Yum");
    return 0;
} 